app/workers/functions.php

```diff
@@ -425,15 +425,24 @@ public function execute(string $trigger, string $projectId, string $executionId,
                 " --workdir /usr/local/src".
                 " ".\implode(" ", $vars).
                 " {$runtime['image']}".
-                " sh -c 'mv /tmp/code.tar.gz /usr/local/src/code.tar.gz && tar -zxf /usr/local/src/code.tar.gz --strip 1 && rm /usr/local/src/code.tar.gz && tail -f /dev/null'"
+                " tail -f /dev/null"
             , '', $stdout, $stderr, 30);
 
-            $executionEnd = \microtime(true);
-    
             if($exitCode !== 0) {
                 throw new Exception('Failed to create function environment: '.$stderr);
             }
 
+            $exitCodeUntar = Console::execute("docker exec ".
+            $container.
```

Comment:
Can we have indentation here - so it aligns with the previous `Console::execute`.